projects/vdk-core/src/vdk/internal/builtin_plugins/run/job_input_error_classifier.py

```diff
@@ -32,7 +32,7 @@ def whom_to_blame(
              errors.ResolvableBy.USER_ERROR if exception was recognized as User Error.
     """
     if isinstance(exception, errors.BaseVdkError) or hasattr(
-        exception, "to_be_fixed_by"
+        exception, errors.ATTR_VDK_RESOLVABLE_BY
```

Comment:
It's better to use error_classifiers.get_exception_resolvable_by(exception) which basically does hasattr(xx) but it's encapsulated 

```
if isinstance(exception, errors.BaseVdkError) or error_classifiers.get_exception_resolvable_by(exception): 
```